bindings/wasm/examples/index.html

```diff
@@ -40,16 +40,25 @@
   <div class="page">
     <div class="container header" style="background-color: #A288B3;">
       <p style="flex: 2;"><a target="_blank" href="https://github.com/elalish/manifold">Manifold</a>CAD </p>
-      <div class="margin" style="position: relative;">
-        <button type="button" id="file" class="blue"><b class="uparrow"></b>&nbsp;&nbsp;<span
+      <div class="margin" style="position: relative; width: 250px;">
```

Comment:
I think the best practice right now is to avoid using px, and use things like em instead?
As px can behave weirdly on mobiles with different dpi.

Reply:
I think it is ok to use px. CSS pixel is not the same as physical pixel. See the docs of [devicePixelRatio](https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio).

Reply:
Oh OK, I haven't done frontend development for years now. Will have a look at the actual result on PC and mobile later today.